wiki/mapping/glossary.md

```diff
@@ -16,232 +16,242 @@ let us know in #mapping-discussion!
-| **Parity** | The mathematical term describing the ‘group’ a block belongs to- within the context of Beat Saber, this refers to the forehand/backhand nature of a block. |
-| **Pattern** | A generic name for a sequence of blocks. |
-| **Paul** | A sequence of inline blocks of the same direction placed at very high precision. This forces the player to hit with a slow continuous swing. This pattern is impossible to score well on. <details><summary>**Example Image**</summary>![Picture of paul](~@images/mapping/paul.jpg)</details>|
-| **Piano Stream** | A sequence of alternating color and direction blocks that progresses horizontally across lanes on the track. <details><summary>**Example Image**</summary>![Picture of piano stream](~@images/mapping/pianostream.png)</details>|
-| **Playtest** | The act of playing a WIP map to check for errors and improvements. Highly recommended for a quality product. |
-| **Performance Points** | The metric determining ranking on the ScoreSaber leaderboards. Abbreviated PP. |
-| **Platform** | The in-game play area and associated track and lights. Also known as [Environment](#e) |
-| **PP** | Abbreviation of Performance Points. |
-| **Precision** | The division of a beat in an editor; used to place notes consistently. Typically 1/1 (one note per beat), 1/2 (one note per half beat), or 1/4 (one note per quarter beat). Also known as [Cursor Precision](#c) |
-| **Precision Placement** | A style of mapping that requires Mapping Extensions which allows the mapper to place blocks outside of the standard 4 x 3 grid. Not recommended for novice mappers. |
-| **Precision Rotation** | A style of mapping that requires Mapping Extensions which allows the mapper to freely rotate blocks (vs. the standard 45 degree rotation). Not recommended for novice mappers. |
+| **Parity** | The mathematical term describing the "group" a [Block](#b) belongs to. Within the context of Beat Saber, this refers to the [Forehand](#f) and [Backhand](#b) nature of a block. |
+| **Pattern** | A generic name for a sequence of [Blocks](#b). |
+| **Paul** | A sequence of inline [Blocks](#b) of the same direction placed at very high [Precision](#p). This forces the player to hit the sequence with a slow continuous swing. This [Pattern](#p) is very difficult to score well on. <details><summary>**Example Image**</summary>![Picture of paul](~@images/mapping/paul.jpg)</details> |
+| **Piano Stream** | A sequence of alternating color and direction [Blocks](#b) that progresses horizontally across [Lanes](#l) on the [Track](#t). <details><summary>**Example Image**</summary>![Picture of piano stream](~@images/mapping/pianostream.png)</details> |
+| **Pickle** | A [Pattern](#p) where a red and blue [Note](#n) are on opposite-handed sides and are hit simultaneously in opposite directions. Also known as [Pickle](#p). See also: [Crossover Scissor](#s). <details><summary>**Example Image**</summary>![Picture of pickle](~@images/mapping/arm-tangle-alt.png)</details> |
```

Comment:
A pickle is indeed also known as a pickle.
Also Crossover Scissor should link to \#c.

```suggestion
| **Pickle** | A [Pattern](#p) where a red and blue [Note](#n) are on opposite-handed sides and are hit simultaneously in opposite directions. See also: [Crossover Scissor](#c). <details><summary>**Example Image**</summary>![Picture of pickle](~@images/mapping/arm-tangle-alt.png)</details> |
```

Reply:
Repetition for emphasis, hah!

Changed the Pickle definition to "Also known as [Crossover Scissor](#c). See also: [Scissor](#s)."